### PR TITLE
Add `ui_params_text` similar to existing `header_text`

### PR DESCRIPTION
**Pull request by xu-hao**
We would like to use a custom javascript function for `componentsSorter`. Currently it is impossible to specify it in `ui_params` since it is converted to JSON. Adding a `ui_params_text` similar to the existing `header_text` would allow us to provide such a specification.

### CODE REVIEW

**Comment by xu-hao** [comment]
@javabrett Thanks for reviewing this PR. I've added an example to flasgger/examples/changelog_090.py.

File: examples/changelog_090.py [hunk 1]
```diff
@@ -51,7 +51,13 @@
     'ui_params': {
         'apisSorter': 'alpha',
         'operationsSorter': 'alpha',
```
Comment (javabrett) [inline examples/changelog_090.py]: I think you'll want to remove this one now, since it is replaced by the function-version below.

**Comment by javabrett** [comment]
Jotting down some thoughts while they are in my head.

My main hesitation with this is whether it can be done without an additional setting.  Maybe it can't, or at least can't without using JSON to go between Python dict and JavaScript object.  The Python `json.JSONEncoder` seems difficult to override, and there's no simple way to distinguish between the quoted string setting and the JS function.

**Comment by javabrett** [comment]
Thanks @xu-hao ☕️ 🍰 .

**Pull request merged**